torchvision/transforms/transforms.py

```diff
@@ -105,7 +105,9 @@ def __repr__(self) -> str:
 
 
 class ToTensor:
-    """Convert a ``PIL Image`` or ``numpy.ndarray`` to tensor. This transform does not support torchscript.
+    """Convert a ``PIL Image`` or ``numpy.ndarray`` to tensor and scale the values accordingly.
```

Comment:
Note that in a bunch of these I had to remove the full dots on those first sentences, because otherwise sphinx would stop after the first `.` when rendering the "preview" which would truncate relevant info.

Reply:
We always mention "PIL Image" without ticks and numpy ndarray as "ndarray". It would worth to keep things similar everywhere ?

Reply:
maybe, but honestly I'd rather do that kind of changes later. The use of ticks doesn't come from this PR